cityofaustin/atd-vz-data
# Replace moment.js with date-fns for VZV

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/12007

I've refactored the entire codebase to use `date-fns` instead of `moment.js` except `SideMapControlDateRange` which uses a component called `DateRangePicker` from this [react-dates](https://github.com/react-dates/react-dates) library that expects/requires a moment object as props, so we need to find something else to use that isn't dependent on moment. Finding a new date range component to use or making our own will take some time so I'm gonna create another issue to address that since this is getting pretty big already.

## Testing
**URL to test:** 
https://deploy-preview-1323--atd-vzv-staging.netlify.app/

**Steps to test:**
1. I think the best way to test is to pull up the [current VZV](https://visionzero.austin.gov/viewer/) and compare everything to this deploy preview to make sure the values, dates, numbers, etc are the same and the functionality is the same.
2. On the dashboard make sure the numbers and graphs are the same.
3. On the map view adjust the dates and test other functionality, make sure its all working.

---
#### Ship list
